doc/source/deployment/deploy.rst

```diff
@@ -58,6 +58,35 @@ To check the deployment progress of the Openstack services:
 
   kubectl get po -n openstack
 
+OpenStack Services
++++++++++++++++++++++++++
```

Comment:
Shouldn't these `+` chars stop at the final `s` above?

Reply:
Yes. done. Apparently I dozed off.